src/DashboardService.Mappers/Db/DbTaskMapper.cs

```diff
@@ -0,0 +1,36 @@
+using LT.DigitalOffice.DashboardService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.Task;
+using LT.DigitalOffice.Kernel.Extensions;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Mappers.Db;
+
+public class DbTaskMapper : IDbTaskMapper
+{
+  private readonly IHttpContextAccessor _httpContextAccessor;
```

Comment:
don't inject it. Pass senderId as a parameter

Reply:
++